crates/astria-sequencer-relayer/src/relayer/write.rs

```diff
@@ -277,16 +301,16 @@ async fn submit_with_retry(
         .max_delay(Duration::from_secs(12))
         .on_retry(
             |attempt: u32, next_delay: Option<Duration>, error: &eyre::Report| {
+                metrics::counter!(crate::metrics_init::CELESTIA_SUBMISSION_FAILURE_COUNT)
```

Comment:
Similar to counting the number of failed celestia submits, would this make sense as a gauge?

Reply:
As above I think having a counter & a guage makes sense.